framework/packages/abstract-core/src/native/account_factory.rs

```diff
@@ -74,10 +83,14 @@ pub enum ExecuteMsg {
         governance: GovernanceDetails<String>,
         // Account name
         name: String,
+        // Optionally specify a base asset for the account
```

Comment:
What's your thought on `CreateAccount(CreateAccountMsg)`?

Reply:
The issue is that we need to verify the governance details, so that's a bit of a hassle when unpacking an repacking the msg

Reply:
We can do the governance verification inside the execute function, that's better

Reply:
The issue is that it's not really compatible with cw_orch !

Reply:
> The issue is that it's not really compatible with cw_orch !

Ah, I was wondering if that would be an issue, thanks. We should add support on those in the future then

Reply:
Yes. However, it's not really that simple as the proc_macro engine doesn't work like that and cannot interpret things that are not defined in the structured that is being parsed